.travis.yml

```diff
@@ -41,3 +42,10 @@ matrix:
       rvm: 2.1
     - gemfile: gemfiles/rails5.1.gemfile
       rvm: 2.2
+
+    - gemfile: gemfiles/rails5.2.gemfile
```

Comment:
FYI can also drop 2.0/2.1

Reply:
done

Reply:
need to remove them from the `rvm` above and bump ruby version in gemspec

Reply:
Not sure how to "bump ruby version in gemspec". What do you mean by that?